oz/core/actions.py

```diff
@@ -65,6 +65,13 @@ def init(project_name):
 def server():
     """Runs the server"""
 
+    if oz.settings["logging"]:
```

Comment:
I don't think this should be necessary, as tornado handles it for you. We just need to add an option for `logging` under https://github.com/dailymuse/oz/blob/develop/oz/core/options.py